runtime/guard-jwt/src/main/java/io/aklivity/zilla/runtime/guard/jwt/internal/JwtGuardHandler.java

```diff
@@ -179,11 +184,23 @@ public long reauthorize(
                 break authorize;
             }
 
-            List<String> roles = Optional.ofNullable(claims.getClaimValue("scope"))
-                .map(s -> s.toString().intern())
-                .map(s -> s.split("\\s+"))
-                .map(Arrays::asList)
-                .orElse(null);
+            List<String> roles = null;
+            String path = (this.roles != null && !this.roles.isEmpty()) ? this.roles : "scope";
```

Comment:
Schema should prevent empty string value if not permitted here and this code can then be simplified to remove the check.
The value of `roles` should be defaulted in the builder during parse to avoid implementing the defaulting logic here where options are being used.